src/dotty/tools/dotc/Compiler.scala

```diff
@@ -55,7 +55,8 @@ class Compiler {
            new Literalize,
            new Getters,
            new ElimByName,
-           new ResolveSuper),
+           new ResolveSuper,
+           new ClassTags),
```

Comment:
Can we swap RresolveSuper and ClassTags. ResolveSuper - erasure - Mixin are sort of a unit that should stay together.